conda/bld.bat

```diff
@@ -1,18 +1,64 @@
 @echo off
-REM This program and the accompanying materials are
-REM made available under the terms of the Eclipse Public License v2.0 which accompanies
-REM this distribution, and is available at https://www.eclipse.org/legal/epl-v20.html
-REM 
-REM SPDX-License-Identifier: EPL-2.0
-REM 
-REM Copyright Contributors to the Zowe Project.
 
+REM Available env vars:
+REM PREFIX - The location you must copy content to be included into the package
+REM PKG_NAME - The name of your package as seen by a user
+REM PKG_VERSION - The version, as seen by a user
+REM PKG_BUILDNUM - The build number, can be used to distinguish patches by the user
 
-if not exist %PREFIX%\share\zowe\app-server\zlux-app-server\defaults\plugins mkdir %PREFIX%\share\zowe\app-server\zlux-app-server\defaults\plugins
-echo '{"identifier":"%PKG_NAME%","pluginLocation":"%PREFIX%/share/zowe/app-server/plugins/%PKG_NAME%/%PKG_VERSION%"}' ^
-     > %PREFIX%\share\zowe\app-server\zlux-app-server\defaults\plugins\%PKG_NAME%.json
 
-if not exist %PREFIX%\share\zowe\app-server\plugins\%PKG_NAME%\%PKG_VERSION% mkdir %PREFIX%\share\zowe\app-server\plugins\%PKG_NAME%\%PKG_VERSION%
-robocopy %SRC_DIR% %PREFIX%\share\zowe\app-server\plugins\%PKG_NAME%\%PKG_VERSION% * /E > nul
+REM /plugins is a primary location for plugins of a component. Secondarily, /components/componentname/plugins
+REM PKG_NAME should be meaningful in the case that multiple plugin dirs (app, cli, apiml) exist within.
+REM If your package includes multiple plugins for a type (such as 2 app framework plugins),
+REM There are different solutions but you may wish to either place common utilities outside the below variables,
+REM While keeping the plugins within these below locations, in a 1-plugin-per-directory scheme
+REM To maintain consistency with other plugins.
+
+set zowe_plugins=%PREFIX%\opt\zowe\plugins
+set app_plugin_dir=%zowe_plugins%\%PKG_NAME%\app-server
+set cli_plugin_dir=%zowe_plugins%\%PKG_NAME%\cli
+set apiml_plugin_dir=%zowe_plugins%\%PKG_NAME%\api-mediation
+REM Special case: zos isnt a component, just a way to make clear what must be put onto zos
+set zos_content_dir=%zowe_plugins%\%PKG_NAME%\zos
+
+REM In the case that the structure of https://github.com/zowe/zowe-install-packaging/issues/1569 is not followed
+REM Then this is assuming it's an app framework plugin.
+if exist %SRC_DIR%\pluginDefinition.json (
+    mkdir %app_plugin_dir%
+    robocopy %SRC_DIR% %app_plugin_dir% * /E > nul
+) else (
+REM Otherwise, if it is following the scheme, here's some hardcoded components that may have content
+    if exist %SRC_DIR%\app-server (
+        mkdir %app_plugin_dir%
+        robocopy %SRC_DIR%\app-server %app_plugin_dir% * /E > nul
+    )
+    if exist %SRC_DIR%\cli (
+        mkdir %cli_plugin_dir%
+        robocopy %SRC_DIR%\cli %cli_plugin_dir% * /E > nul
+    )
+    if exist %SRC_DIR%\api-mediation (
```

Comment:
According to https://github.com/zowe/zowe-install-packaging/issues/1569

You titled this "apiml"